src/data.js

```diff
@@ -2064,6 +2064,46 @@ const pages = [
       'YouTuber',
     ],
   },
+  {
+    // Your name
+    name: 'Olli Rautiainen',
+    // Short description
+    description: 'human, bird, banana, cat daddy',
+    // URL to your /uses page
+    url: 'https://github.com/0LL1',
```

Comment:
The URL needs to be a "uses" page, having it in a gist with your gear/software is fine too if you don't have a site.